hack/make-rules/test-integration.sh

```diff
@@ -67,7 +67,6 @@ runTests() {
   kube::etcd::start
   kube::log::status "Running integration test cases"
 
-  KUBE_RACE="-race"
```

Comment:
This will change how we run integration tests right?
If we need it (and it seems we do), please create a separate PR with it and send it to test-infra folks.
We should merge those two things in a single PR.

Reply:
We don't really need that change - changed it just because I noticed it. It seems to be a no-op as in the `make test` command below we're anyway setting KUBE_RACE="". IIUC this line doesn't serve any purpose.

Reply:
Then let's not remove it in this PR.

Reply:
Done